workers/generic-worker/release.sh

```diff
@@ -8,7 +8,8 @@
 # exit in case of bad exit code
 set -e
 
-OFFICIAL_GIT_REPO='git@github.com:taskcluster/generic-worker'
+# TODO: is this scrpit used at all?
+OFFICIAL_GIT_REPO='git@github.com:taskcluster/taskcluster'
```

Comment:
Good spot, indeed we can delete this script now! 🥳

Reply:
cool! I'll drop it